content/docs/latest/manual-installation_283118363.md

```diff
@@ -378,6 +379,75 @@ That directory should contain all the files necessary to run Hive. You can run i
 
 From now, you can follow the steps described in the section Installing Hive from a Tarball
 
+## Installing with old version hadoop(>=3.1.0)
+
+Although we normally require hive4 to rely on a 
+hadoop 3.3.6 cluster environment. 
+However, in practice, in an ON YARN environment,
+we can package all the hadoop related dependencies into 
+tez&hive so that they do not need to rely on the lib 
+of the original hadoop cluster environment at runtime. 
+In this way, we can run HIVE4 in a lower version of hadoop, 
+provided that the base APIs of the hadoop 3.x series are common to 
+each other.
+
+The steps are as follows:
+
+1.Download the high version of the Hadoop package, unzip it, and then set the hadoop_home finger of the env script in HIVE4 to the path where the high version of hadoop is unzipped.
+
+2.Compile TEZ to get tez.tar.gz which contains all hadoop related dependencies, 
+first extract it on the physical machine where HIVE is deployed and configure the TEZ_HOME in HIVE to point to it, 
+then place tez.tar.gz in a path in hdfs.
+
+```shell
+[root@hmsclient01 opt]# ll
```

Comment:
what is the `hmsclient01` and `opt` folder, is that from docker container?

Reply:
This is just a directory in our production environment; I'm just illustrating how to install.